packaging/docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,56 @@
+ARG BASE_IMAGE=ubuntu:18.10
+FROM ${BASE_IMAGE} AS grafana-builder
+
+ARG GRAFANA_TGZ="grafana-latest.linux-x64.tar.gz"
+
+COPY ${GRAFANA_TGZ} /tmp/grafana.tar.gz
+
+RUN mkdir /tmp/grafana && tar xfz /tmp/grafana.tar.gz --strip-components=1 -C /tmp/grafana
+
+FROM ${BASE_IMAGE}
+
+EXPOSE 3000
```

Comment:
@xlson By builder part of the image, do you mean the part of the Dockerfile where building takes place? Does that make any practical difference to the result? I put it at the beginning since I tend to do that by convention, since it's easier to see the metadata of the image (also `LABEL` f.ex.).

Reply:
Makes sense. I'm good with that.